src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ProcessorService.cs

```diff
@@ -79,10 +74,7 @@ public async Task ProcessStage2(ImportMessage message)
         {
             var subjectData = await GetSubjectDataFromMainDataFile(message);
 
-            await ProcessSubject(message,
-                DbUtils.CreateStatisticsDbContext(),
-                DbUtils.CreateContentDbContext(),
```

Comment:
This leaves the method `Data.Importer.Utils.DbUtils.CreateContentDbContext()` unused.